triage/2020.md

```diff
@@ -0,0 +1,21 @@
+# 2020 Triage Logs
+
+These are in reverse chronological order.
+
+## 2020-05-26 - njn
```

Comment:
Maybe would be good to note "since" on the first one? (i.e. give a range)

I guess for future ones it won't really matter since it'll be implicit from the previous one.

Reply:
Yeah, I don't think it's important.